src/main.js

```diff
@@ -25,6 +27,56 @@ function displayBooks() {
   }
 }
 
+/**
+ * A function that gets book information from user inputted ISBN via API
+ * 
+ */
+ function regByISBN() {
+  let isbn;
+  const readline = require("readline");
+
+  const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+  });
+
+  rl.question("Enter book ISBN: ", function(isbn) {
+    console.log('\n');
+    console.log(`Registering ISBN: ${isbn}...`);
+    rl.close();
+
+    const openLibraryURL = 'https://openlibrary.org';
+    const bookEndPoint = '/isbn/';
+    const apiSuffix = '.json';
+    // const sampleISBN = '9780316212366'
+
+    let bookURL = openLibraryURL + bookEndPoint + isbn + apiSuffix;
+    console.log(bookURL);
+    
+      fetch(bookURL)
+        .then(res => res.json()) // .then OR could be async+await
+        .then(book => {
+
+          const authorEndPointWithID = book.authors[0].key;
+          const bookTitle = book.title;
+          let authorURL = openLibraryURL + authorEndPointWithID + apiSuffix;
+
+          fetch(authorURL)
```

Comment:
Rather than nesting the promises, you should return them and chain them together.

```suggestion
fetch(bookURL)
  .then(res => res.json())
  .then(book =>
    // do stuff with book
    return fetch(authorURL)
  .then(res => res.json())
  .then(author => {
    // do stuff with author
  })
```

Reply:
Ah okay!

Reply:
Noice